homeassistant/components/synology_dsm/translations/de.json

```diff
@@ -34,7 +35,8 @@
                     "password": "Passwort",
                     "port": "Port",
                     "ssl": "Verwenden Sie SSL/TLS, um eine Verbindung zu Ihrem NAS herzustellen",
-                    "username": "Benutzername"
+                    "username": "Benutzername",
+                    "verify_ssl": "SSL-Zertifikat \u00fcberpr\u00fcfen"
```

Comment:
still having de.json

Reply:
yes, because of rule 1 in [Contributing Translation](https://developers.home-assistant.io/docs/translations/#rules) I thought it is ok to already add the German translation

> 1. Only native speakers should submit translations.

Reply:
Yes, but should translate via Localise.com, see the intro of the guide